stellar-contract-env-common/src/env.rs

```diff
@@ -183,6 +183,13 @@ macro_rules! call_macro_with_all_host_functions {
                 {"_", fn compute_hash_sha256(x:Object) -> Object}
                 {"0", fn verify_sig_ed25519(x:Object, k:Object, s:Object) -> RawVal}
             }
+
+            mod account "a" {
+                {"_", fn get_low_threshold(a:Object) -> RawVal}
```

Comment:
Could you add an `account_` prefix to these? We're using an object name prefix on host fns for most object types.

Reply:
Done.